tests/rust/build.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import shlex
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--dry-run", action="store_true")
+parser.add_argument("--verbose", action="store_true")
+parser.add_argument("--release", action="store_true")
+
+args = parser.parse_args()
+
+SYSTEMS = ['wasm32']
+VERSIONS = ['wasip1'] # + ['wasip2', 'wasip3']
+
+def compute_target(system, version):
+    return f"{system}-{version}"
+
+def compute_build_target(system, version):
+    if version == 'wasip3':
+        # wasm32-wasip3 triple not yet supported.
+        return compute_target(system, 'wasip2')
+    return compute_target(system, version)
+
+BASE_DIR = Path(__file__).parent
```

Comment:
That could be a cli argument with the default value set to `Path(__file__).parent` for local executions.